civis/resources/_resources.py

```diff
@@ -421,10 +438,24 @@ def parse_swagger(swagger, api_version, resources):
 
 
 @lru_cache(maxsize=4)
-def get_swagger_spec(api_key, user_agent, api_version):
+def get_swagger_spec(api_key, user_agent=None, api_version="1.0"):
+    """Download the Civis API specification.
+
+    Parameters
+    ----------
+    api_key : str
+        Your API key obtained from the Civis Platform. If not given, the
+        client will use the :envvar:`CIVIS_API_KEY` environment variable.
+    user_agent : str
+        The user agent used in the request to get api endpoint specification.
```

Comment:
> Why would anyone want to change this from the default?

We use this function in the library and pass an altered User-Agent with `"civis-python/[VERSION]"`.  If you were a bit perplexed by the parameter, users will be too.  I'll make a small refactor to just default to the `"civis-python/[VERSION]"` User-Agent and remove the parameter.